### PR TITLE
Peep Pickup Game Action

### PR DESCRIPTION
**Pull request by duncanspumpkin**
Guest and Staff pickup back into one simple game action.
Tested seems to work.

4
3

### CODE REVIEW

File: src/openrct2/Game.cpp [hunk 1]
```diff
@@ -96,8 +96,8 @@ static GAME_COMMAND_CALLBACK_POINTER * const game_command_callback_table[] = {
     nullptr,
     nullptr,
     nullptr,
-    game_command_callback_pickup_guest,
-    game_command_callback_pickup_staff
+    nullptr,
+    nullptr
```
Comment (Gymnasiast) [inline src/openrct2/Game.cpp]: Now this table is empty, might as well get rid of it if it isn't too much trouble.

Comment (duncanspumpkin) [reply src/openrct2/Game.cpp]: I thought about doing that but then decided I'd make a second pr for that as I've a feeling it won't be as simple as just deleting this bit

**Comment by duncanspumpkin** [comment]
There is a bug that prevents networked games from letting you drop peeps.

**Comment by ZehMatt** [comment]
There seems to be an issue with canceling, this is done locally only so the moment you cancel the pickup the other clients will still see the peep in a pickup state and immediately desync.

**Comment by duncanspumpkin** [comment]
I'm going to change it so that I pass the peep owner id instead of using the game action player id.

**Comment by duncanspumpkin** [comment]
Found a small mistake in the pickup code. Not ready for merge.

**Pull request merged**